static/app/views/dashboards/releasesSelectControl.tsx

```diff
@@ -85,7 +85,7 @@ function ReleasesSelectControl({
           ],
         },
       ]}
-      onChange={opts => setActiveReleases(opts.map(opt => opt.value))}
+      onChange={opts => setActiveReleases(opts.map(opt => opt.value as string))}
```

Comment:
This started complaining with a TS issue, but because this hasn't changed at all I've just silenced it by asserting it's a string.

Reply:
@narsaynorath We had the exact same problem [here](https://github.com/getsentry/sentry/pull/45997). cc @vuluongj20 it seems that styling a compact select breaks type inference

Reply:
I think the PR you meant to link was https://github.com/getsentry/sentry/pull/45855/, but that's interesting!

Reply:
Ah yes, copy paste went wrong 😅